demisto_sdk/commands/create_artifacts/tests/contet_creator_test.py

```diff
@@ -0,0 +1,31 @@
+import filecmp
+from demisto_sdk.commands.create_artifacts.content_creator import *
+
+
+def test_copy_dir_files():
```

Comment:
test is unclear, what are you trying to test here?

Reply:
added in the test description.